doc/api/process.markdown

```diff
@@ -218,7 +221,7 @@ Note:
   the terminal mode before exiting with code `128 + signal number`. If one of
   these signals has a listener installed, its default behavior will be removed
   (Node.js will no longer exit).
-- `SIGPIPE` is ignored by default, it can have a listener installed.
+- `SIGPIPE` is ignored by default. It can have a listener installed.
```

Comment:
`git blame` says that line was introduced in the docs in October 2013 with https://github.com/nodejs/node/commit/155df9ca7. The behavior predates the documentation of it, but I don't know by how much.